package.json

```diff
@@ -68,11 +67,11 @@
     "@18f/identity-components": "^2.0.0",
     "@18f/identity-design-system": "^7.1.0",
     "@18f/identity-i18n": "^1.0.1",
-    "@babel/core": "^7.12.9",
-    "@babel/preset-env": "^7.12.7",
-    "@babel/preset-react": "^7.22.5",
-    "@babel/preset-typescript": "^7.18.6",
-    "babel-loader": "^8.2.4",
+    "@babel/core": "^7.23.2",
+    "@babel/preset-env": "^7.23.2",
+    "@babel/preset-react": "^7.22.15",
+    "@babel/preset-typescript": "^7.23.2",
+    "babel-loader": "^9.1.3",
```

Comment:
Breaking changes are dropping support for Webpack 4 and older versions of Node. Should be fine.

https://github.com/babel/babel-loader/releases/tag/v9.0.0